content/en/docs/5.configuration-glossary/2.configuration-build.md

```diff
@@ -1,30 +1,30 @@
 ---
```

Comment:
Same as the previous PR, this file is `content/en` so it should not be translated

Reply:
Wow 😅,
It is so strange and I did not repair in it,
I am fixing it as soon as possible, an effort
will be did to avoid this happen in the next
pull request 😊.